#! /bin/sh /usr/share/dpatch/dpatch-run
## 99-unnamed.dpatch by  <abi@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: disable zabbix internal log-rotation, we have logrotate.

@DPATCH@
--- log.c	2007-03-30 20:10:25.000000000 +0200
+++ zabbix-1.1.7/src/libs/zbxlog/log.c	2007-03-30 20:11:06.000000000 +0200
@@ -148,22 +148,6 @@
 		fclose(log_file);
 
 
-		if(stat(log_filename,&buf) == 0)
-		{
-			if(buf.st_size > MAX_LOG_FILE_LEN)
-			{
-				strscpy(filename_old,log_filename);
-				zbx_strlcat(filename_old,".old",MAX_STRING_LEN);
-				if(rename(log_filename,filename_old) != 0)
-				{
-/*					exit(1);*/
-				}
-			}
-			if(old_size > buf.st_size)
-				redirect_std(log_filename);
-
-			old_size = buf.st_size;
-		}
 	}
 	else
 	{
